src/analyze/analyze.c

```diff
@@ -1551,6 +1553,29 @@ static int dump_syscall_filters(int argc, char *argv[], void *userdata) {
 }
 #endif
 
+static int dump_timespan(int argc, char *argv[], void *userdata) {
+        int r;
+        char **input_timespan;
+        usec_t usec_magnitude = 1, output_usecs;
+        char ft_buf[FORMAT_TIMESPAN_MAX];
+
+        STRV_FOREACH(input_timespan, strv_skip(argv, 1)) {
+            r = parse_time(*input_timespan, &output_usecs, usec_magnitude);
+            if (r < 0) {
+                    return log_error_errno(r, "Failed to parse time span '%s': %m", *input_timespan);
+            }
+
+            printf("Original: %s\n", *input_timespan);
+            printf("      μs: %" PRIu64 "\n", output_usecs);
+            printf("   Human: %s\n", format_timespan(ft_buf, sizeof(ft_buf), output_usecs, usec_magnitude));
+
+            if (*(input_timespan + 1))
```

Comment:
I think `if (input_timespan[1])` is a tiny bit more readable, but doesn't matter